app/routes/shop.py

```diff
@@ -114,4 +118,27 @@ def upload_stamp_image():
   response.raise_for_status()
   data = response.json()
 
-  return { 'data':  data }
\ No newline at end of file
+  return { 'data':  data }
+
+@bp.route('/upload', methods=['POST'])
+@jwt_required()
+def upload():
+  if 'file' not in request.files:
+    return { 'msg': 'No file included in request' }, 422
+  # If the user does not select a file, the browser submits an
+  # empty file without a filename.
+  file = request.files.get('file')
+  if file.filename == '':
+      return { 'msg': 'Submitted an empty file' }, 422
+  import uuid
+  s3_key = str(get_current_user().id) + '_' + str(uuid.uuid4())
+  contents = file.read()
+  bucket = os.environ.get('S3_BUCKET_NAME')
+  logging.error (f"Content Type {file.content_type}")
```

Comment:
let's remove this

Reply:
I'm assuming it's only the `logging` statement we're removing, because removing the other visible lines would break the. endpoint.

Reply:
yeah just the logging statement. usually i just mean the line i directly comment on. ill specify when i mean any surrounding lines also